spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/ClusterMonitorType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.hazelcast;
+
+/**
+ * Enumeration of Hazelcast Cluster Monitor Types
+ *
+ * @author Eren Avsarogullari
+ * @since 1.0.0
+ * @see org.springframework.integration.hazelcast.inbound.HazelcastClusterMonitorMessageProducer
+ */
+public enum ClusterMonitorType {
+
+	MEMBERSHIP, DISTRIBUTED_OBJECT, MIGRATION, LIFECYCLE, CLIENT;
```

Comment:
I think we must make from this class' JavaDoc reference links to these events.
For example I don't know what they mean, so I'd like reading anything about them.
Thank